docs/data/material/getting-started/templates/checkout/Checkout.tsx

```diff
@@ -55,68 +117,297 @@ export default function Checkout() {
+          lg={4}
+          sx={{
+            display: { xs: 'none', md: 'flex' },
+            flexDirection: 'column',
+            backgroundColor: 'background.paper',
+            borderRight: { sm: 'none', md: '1px solid' },
+            borderColor: { sm: 'none', md: 'divider' },
+            alignItems: 'start',
+            pt: 4,
+            px: 10,
+            gap: 4,
+          }}
+        >
+          <Box
+            sx={{
+              display: 'flex',
+              alignItems: 'end',
+              height: 150,
+            }}
+          >
+            <Button
+              startIcon={<ArrowBackRoundedIcon />}
+              component="a"
+              href="/material-ui/getting-started/templates/landing-page/"
+              sx={{ ml: '-8px' }}
+            >
+              Back to
+              <img
+                src={
+                  'https://assets-global.website-files.com/61ed56ae9da9fd7e0ef0a967/61f12e6faf73568658154dae_SitemarkDefault.svg'
+                }
+                style={logoStyle}
+                alt="Sitemark's logo"
+              />
+            </Button>
+          </Box>
+          <Box
+            sx={{
+              display: 'flex',
+              flexDirection: 'column',
+              flexGrow: 1,
+              width: '100%',
+              maxWidth: 500,
+            }}
+          >
+            <Info />
+          </Box>
+        </Grid>
+        <Grid
+          item
+          sm={12}
+          md={7}
+          lg={8}
+          sx={{
+            display: 'flex',
+            flexDirection: 'column',
+            maxWidth: '100%',
+            width: '100%',
+            backgroundColor: { xs: 'transparent', sm: 'background.default' },
+            alignItems: 'start',
+            pt: { xs: 2, sm: 4 },
+            px: { xs: 2, sm: 10 },
+            gap: { xs: 4, md: 8 },
+          }}
+        >
+          <Box
+            sx={{
+              display: 'flex',
+              justifyContent: { sm: 'space-between', md: 'flex-end' },
+              alignItems: 'center',
+              width: '100%',
+              maxWidth: { sm: '100%', md: 600 },
+            }}
+          >
+            <Box
+              sx={{
+                display: { xs: 'flex', md: 'none' },
+                flexDirection: 'row',
+                width: '100%',
+                justifyContent: 'space-between',
+              }}
+            >
+              <Button
+                startIcon={<ArrowBackRoundedIcon />}
+                component="a"
+                href="/material-ui/getting-started/templates/landing-page/"
+                sx={{ alignSelf: 'start' }}
+              >
+                Back to
+                <img
+                  src={
+                    'https://assets-global.website-files.com/61ed56ae9da9fd7e0ef0a967/61f12e6faf73568658154dae_SitemarkDefault.svg'
+                  }
+                  style={logoStyle}
+                  alt="logo of sitemark"
```

Comment:
```suggestion
                  alt="Sitemark's logo"
```

You could also add it on a variable given we repeat it a few times